src/Integration.UnitTests/PathHelperTests.cs

```diff
@@ -100,6 +100,14 @@ public void PathHelper_CalculateRelativePath()
                 fromPath: @"X:\file with spaces (1).ext",
                 toPath: @"X:\file with spaces (2).ext"
             );
+
+            // Non canonical paths (contains . and ..)
+            VerifyCalculateRelativePath
```

Comment:
Test hole. Covering that `CalculateRelativePath` is aware of non-canonical paths.